firestartr-bootstrap/README.md

```diff
@@ -16,31 +16,62 @@ You'll need to install on your local machine:
 - [**dagger**](https://docs.dagger.io/install) (v0.18.5+)
 - [**kind**](https://kind.sigs.k8s.io/docs/user/quick-start#installation) (v0.24.0+).
 
+Create a kind cluster using the following command:
+
+```shell
+kind create cluster
+```
+
+Using `docker ps`, note the port that kind is using to expose the Kubernetes API server. You will need it later when launching the bootstrap.
+
 #### 1.2 AWS requirements
 
 The following AWS Parameter Store parameters are required:
-- `/firestartr/<org-name>/fs-<org-name>-admin/pem`
-- `/firestartr/<org-name>/fs-<org-name>-admin/app-id`
-- `/firestartr/<org-name>/fs-<org-name>-admin/installation-id`
-- `/firestartr/<org-name>/prefapp-bot-pat`
+
+- `/firestartr/<customer>/fs-<customer>/pem`
+- `/firestartr/<customer>/fs-<customer>/app-id`
+- `/firestartr/<customer>/fs-<customer>/<org>/installation-id`
+- `/firestartr/<customer>/fs-<customer>-admin/pem`
+- `/firestartr/<customer>/fs-<customer>-admin/app-id`
+- `/firestartr/<customer>/fs-<customer>-admin/<org>/installation-id`
+- `/firestartr/<customer>/fs-<customer>-checks/pem`
+- `/firestartr/<customer>/fs-<customer>-checks/app-id`
+- `/firestartr/<customer>/fs-<customer>-checks/<org>/installation-id`
+- `/firestartr/<customer>/fs-<customer>-state/pem`
+- `/firestartr/<customer>/fs-<customer>-state/app-id`
+- `/firestartr/<customer>/fs-<customer>-state/<org>/installation-id`
+- `/firestartr/<customer>/fs-<customer>-import/pem`
+- `/firestartr/<customer>/fs-<customer>-import/app-id`
+- `/firestartr/<customer>/fs-<customer>-import/<org>/installation-id`
+- `/firestartr/<customer>/fs-<customer>-argocd/pem`
+- `/firestartr/<customer>/fs-<customer>-argocd/app-id`
+- `/firestartr/<customer>/fs-<customer>-argocd/<org>/installation-id`
+
+The following AWS Parameter Store are going to be generated and/or uploaded ( pushed ) by the Bootstrap process
+
+- `/firestartr/<customer>/prefapp-bot-pat`: Personal Access Token for the Prefapp Bot user
+- `/firestartr/<customer>/firestartr-cli-version`: Version of the Firestartr CLI to set as the default in the organization
+- `/firestartr/<customer-name>/github-webhook/secret`: Secret for the GitHub Webhook
 
 ### 2. Bootstrap File
 
 ```yaml
-# Bootstrapfile.yaml
+# BootstrapFile.yaml
 ---
 org: <github-org>
+env: <env>  # either "pre" or "pro"
+customer: <customer-name>  # name used for the org internally, within the parameter store
```

Comment:
Name this property so that it is as aligned as possible.


```suggestion
customer: <customer>  # name used for the org internally, within the parameter store
```